javatests/arcs/android/host/AndroidEntityHandleManagerTest.kt

```diff
@@ -176,14 +184,13 @@ class AndroidEntityHandleManagerTest : LifecycleOwner {
     }
 
     @Test
-    fun singletonHandle_writeInOnSyncNoDesync() = runBlocking {
+    fun singletonHandle_writeInOnSyncNoDesync() = runBlocking<Unit> {
```

Comment:
Is <Unit> neccessary?

Reply:
ah, it will be when the main PR lands.. since `store` will return a Job that can be joined-on to wait for completion